engine/engine/content/builtins/scripts/mobdebug.lua

```diff
@@ -330,7 +330,7 @@ local function stack(start)
     if not source then break end
 
     local src = source.source
-    if src:find("[@=]") == 1 then
+    if src:find("@") == 1 then
```

Comment:
We can now safely revert this patch since we no longer provide filenames starting with = (see script_module.cpp)